frontend/src/metabase/components/EmptyState/EmptyState.tsx

```diff
@@ -87,8 +87,8 @@ const EmptyState = ({
     {/* TODO - we should make this children or some other more flexible way to
       add actions
       */}
-    <EmptyStateFooter>
-      <EmptyStateActions>
+    <Flex>
+      <EmptyStateActions className={S.EmptyStateFooter}>
```

Comment:
The class targets `EmptyStateActions` instead because I'm using `:not(:empty)` here.